startupshutdown/startupshutdown.go

```diff
@@ -1,73 +1,81 @@
 package startupshutdown
 
 import (
-	"log"
-	"os"
-	"os/exec"
+	"fmt"
 	"strings"
 
+	"github.com/google/uuid"
 	"github.com/mholt/caddy"
+	"github.com/mholt/caddy/onevent/hook"
 )
 
 func init() {
 	caddy.RegisterPlugin("startup", caddy.Plugin{Action: Startup})
 	caddy.RegisterPlugin("shutdown", caddy.Plugin{Action: Shutdown})
 }
 
-// Startup registers a startup callback to execute during server start.
+// Startup is an "alias" for on startup.
```

Comment:
I think we should add to this comment rather than replace.  I would suggest

    // Startup (an alias for 'on startup' ) registers a startup callback to execute during server start.